charts/dependentapi-simple-operator/templates/rbac.yaml

```diff
@@ -92,6 +96,11 @@ rules:
     resources: [components]
     verbs: [list, watch, patch, get]
 
+  # Application: watching & handling for the custom resource we declare.
+  - apiGroups: [oda.tmforum.org]
+    resources: [apis]
+    verbs: [list, watch, get, patch]
```

Comment:
"patch" needed?

Reply:
API PATCH is not needed for dependent API.
I will remove the PATCH from the list and raise a new PR.